### PR TITLE
new links

### PR DESCRIPTION
**Pull request by sahehb**

### CODE REVIEW

File: mevislab.github.io/content/examples/image_processing.md [hunk 1]
```diff
@@ -13,6 +13,6 @@ menu:
 ## Image Processing Examples:
 [Arithmetic Operations](/examples/image_processing/example1/)  
 [Masking Images](/examples/image_processing/example2/)  
-[Region Growing (Segmentation)](/examples/image_processing/example3/) 
-[Subtract 3D objects](/examples/image_processing/example4/) 
+[Region Growing (Segmentation)](/examples/image_processing/example3/)  
+[Subtract 3D objects](/examples/image_processing/example4/)
```
Comment (mrbean-bremen) [inline mevislab.github.io/content/examples/image_processing.md:17]: Is there only a space added and line ends?

Comment (sahehb) [reply mevislab.github.io/content/examples/image_processing.md:17]: yes, without spaces, linebreaks in HTML are missing, strange behaviour :(

Comment (mrbean-bremen) [reply mevislab.github.io/content/examples/image_processing.md:17]: Ah, true - this is standard markdown syntax, 2 spaces denote a line break

Comment (sahehb) [reply mevislab.github.io/content/examples/image_processing.md:17]: cant we leave it like this? Its standard md and works :)

**Pull request merged**